Libraries/PeriphDrivers/Source/SYS/sys_ai85.c

```diff
@@ -377,18 +383,13 @@ int MXC_SYS_Clock_Select(mxc_sys_system_clock_t clock)
         break;
 
     case MXC_SYS_CLOCK_EXTCLK:
-        // Enable HIRC clock
-        // if(!(MXC_GCR->clkctrl & MXC_F_GCR_CLKCTRL_EXTCLK_EN)) {
-        //     MXC_GCR->clkctrl |=MXC_F_GCR_CLKCTRL_EXTCLK_EN;
-
-        //     // Check if HIRC clock is ready
-        //     if (MXC_SYS_Clock_Timeout(MXC_F_GCR_CLKCTRL_EXTCLK_RDY) != E_NO_ERROR) {
-        //         return E_TIME_OUT;
-        //     }
-        // }
-
-        // Set HIRC clock as System Clock
-        // MXC_SETFIELD(MXC_GCR->clkctrl, MXC_F_GCR_CLKCTRL_SYSCLK_SEL, MXC_S_GCR_CLKCTRL_SYSCLK_SEL_EXTCLK);
+        // No EXT_CLK "RDY" bit for AI85 so we enable every time
+        err = MXC_SYS_ClockSourceEnable(MXC_SYS_CLOCK_EXTCLK);
+        if (err)
```

Comment:
Add brackets.

Reply:
> Did we get a customer request for the example? If so, consider adding it for other micros which support external clocks as well.

I haven't added the example universally yet because of the testing overhead associated with it.  We've got the internal SGS group looking at this specifically for the ME15, but I don't think I'll have time to port + test the example universally yet.  Todo in a future PR

Reply:
> Add brackets.

-.-

https://github.com/Analog-Devices-MSDK/msdk/pull/521/commits/0240928ae0077cb4651bbe52a4c526f63a71a2d0